src/collection/tx_data/collector.py

```diff
@@ -0,0 +1,282 @@
+"""
+Class for collection of transaction data from Solana chain
+"""
+import asyncio
```

Comment:
Pls sort the imports :pray:

Reply:
done. it s nice to have it alphabetically sorted, but totally unnecessary out of scope that is suggested by pylint.

Reply:
Yeah, sure. It's just esthetics.